sdk/src/rest/composable.ts

```diff
@@ -31,11 +31,10 @@ export const rest = (config: Partial<RestConfig> = {}) => {
 				}
 
 				// we need to use THIS here instead of client to access overridden functions
-				if ('getToken' in this) {
+				if ('getToken' in this && 'Authorization' in options.headers === false) {
```

Comment:
This line will error if `options.headers` is `undefined`
![image](https://github.com/user-attachments/assets/de88345f-04b7-43b4-9ccc-9f2f4d8d2b6f)

```suggestion
				if ('getToken' in this && (!options.headers || 'Authorization' in options.headers === false)) {
```

Reply:
We force `options.headers` to always be defined a couple of lines above it tho

https://github.com/directus/directus/blob/5e9e4b79dff85fc9e76d0305b6f3c11b1f5f7635/sdk/src/rest/composable.ts#L22-L24

Reply:
fair point did not see that in the PR diff 😄